src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -133,6 +133,7 @@ CATCH_RETURN()
                                                                _In_ IDWriteTextRenderer* renderer,
                                                                FLOAT originX,
                                                                FLOAT originY) noexcept
+try
 {
     const auto drawingContext = static_cast<const DrawingContext*>(clientDrawingContext);
     _formatInUse = drawingContext->useItalicFont ? _fontRenderData->ItalicTextFormat().Get() : _fontRenderData->DefaultTextFormat().Get();

```

Comment:
This is where I'll be adding `bold` and `bold-italic`. @DHowett is this a feasible plan to you? It works, of course. Just want to know if it's a good practice.

Reply:
On my other branch, it looks like this:

```cpp
if (drawingContext->useBoldFont && drawingContext->useItalicFont)
{
    _formatInUse = _fontRenderData->BoldItalicTextFormat().Get();
    _fontInUse = _fontRenderData->BoldItalicFontFace().Get();
}
else if (drawingContext->useBoldFont)
{
    _formatInUse = _fontRenderData->BoldTextFormat().Get();
    _fontInUse = _fontRenderData->BoldFontFace().Get();
}
else if (drawingContext->useItalicFont)
{
    _formatInUse = _fontRenderData->ItalicTextFormat().Get();
    _fontInUse = _fontRenderData->ItalicFontFace().Get();
}
else
{
    _formatInUse = _fontRenderData->DefaultTextFormat().Get();
    _fontInUse = _fontRenderData->DefaultFontFace().Get();
}
```